javascript/ajax.js

```diff
@@ -730,7 +730,7 @@ function fetch_jobs() {
 	    oTable.$("td.editableStatus").editable("./scripts/db_update2.pl", {
 		indicator  : "Saving...",
 		tooltip    : "Click to edit...",
-		data       : "{'SEMAPHORED':'SEMAPHORED','READY':'READY','RUN':'RUN','DONE':'DONE'}",
+		data       : "{'SEMAPHORED':'SEMAPHORED','READY':'READY','RUN':'RUN','DONE':'DONE','PASSED_ON':'PASSED_ON'}",
```

Comment:
Can the value be changed to PASSED<space>ON rather than PASSED_ON? Will it break anything downstream?

Reply:
PASSED_ON is hard coded as the status in the database, so this needs to be preserved. https://github.com/Ensembl/ensembl-hive/blob/68ee32fb39676d070f77baaf7d532396d6f76bf3/sql/tables.mysql#L334

Reply:
Ok. Fair enough.